src/Moat/Pretty/Kotlin.hs

```diff
@@ -34,7 +34,7 @@ prettyKotlinData = \case
   MoatNewtype {..} ->
     ""
       ++ prettyAnnotations newtypeAnnotations
-      ++ "inline class "
+      ++ prettyNewtypeDeclaration newtypeDecl
```

Comment:
I think we could check here to see if the value class is used here, and then add JvmInline to the Annotations if it isn't already present.